code/game/objects/structures/secure_safe.dm

```diff
@@ -43,11 +74,14 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/secure_safe, 32)
 		It is made out of the same material as the station's Black Box and is designed to resist all conventional weaponry. \
 		There appears to be a small amount of surface corrosion. It doesn't look like it could withstand much of an explosion.\
 		It remains quite flush against the wall, and there only seems to be enough room to fit something as slim as an ID card."
```

Comment:
you need to update this description if you're not putting it on the wall anymore.